types/json-pointer/index.d.ts

```diff
@@ -1,82 +1,174 @@
-// Type definitions for json-pointer 1.0
-// Project: https://www.npmjs.org/package/json-pointer
+// Type definitions for json-pointer 0.6
```

Comment:
Nonblocking: This might not be a great idea, even if it's accurate, since we'll publish this on the `0.*` line and it'll basically never get picked up except by people who know to go find it since `latest` gets pegged to the highest major version. Your call.

Reply:
What should I do here then? Publish it as 1.0?

Reply:
That's what I'd do

Reply:
done